src/main/java/DukeTest.java

```diff
@@ -0,0 +1,78 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class DukeTest {
+
+    @Test
+    public void Deadline_Wrong_Format1_exceptionThrown() {
```

Comment:
I think test method naming should be in camelCase? Likewise for the other test methods names